tchannel/testing/vcr/server.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import absolute_import
+
+from tornado import gen
+
+from tchannel.tornado import TChannel
+
+from . import types
+
+
+class FakeServer(object):
+    """A fake TChannel server to record or replay requests."""
+
+    def __init__(self, cassette, real_peer):
+        self.cassette = cassette
+        self.real_peer = real_peer
+        self.tchannel = TChannel('fake-server')
+
+        self.tchannel.register(self.tchannel.FALLBACK)(self.handle_request)
```

Comment:
self.tchannel.register(self.tchannel.FALLBACK, handler=self.handle_request) . The way you are using is to use register to return a decorator. Not quite human readable in this way.